src/main/java/webserver/RequestHandler.java

```diff
@@ -25,25 +27,46 @@ public void run() {
 
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
             BufferedReader br = new BufferedReader(new InputStreamReader( in, StandardCharsets.UTF_8));
-            StringBuilder stringBuilder = new StringBuilder();
+            DataOutputStream dos = new DataOutputStream(out);
 
-            String line = null;
-            while (!(line = br.readLine()).equals("")) {
-                stringBuilder.append(line).append("\n");
-            }
-            stringBuilder.append("\n");
+            String requestHead = readRequestHead(br);
```

Comment:
```suggestion
            List<String> requestHead = readRequestHead(br);            
```
\n 으로 구분하는 것보다 이게 좋지 않을까요?